# Importing Libraries

In [2]:
import pandas as pd
import matplotlib as plt

# Loading the Dataset

Data is downloaded to the `/data` subdirectory of the root project folder. We will use `pandas` to create a data frame with this data. 

In [8]:
data = pd.read_csv('data/AirPassengers.xls')
data.head()

Month  #Passengers
0  1949-01          112
1  1949-02          118
2  1949-03          132
3  1949-04          129
4  1949-05          121

We see that the dataset looks relatively simple. It contains monthly data about the number of air passengers. Let's see more detailed description of the data using `pandas.describe()`:

In [9]:
data.describe()

#Passengers
count   144.000000
mean    280.298611
std     119.966317
min     104.000000
25%     180.000000
50%     265.500000
75%     360.500000
max     622.000000

Having taken a better look we can note down a couple observations:
 - There are 144 observations
 - The distribution is centered around 280
 - The distribution is skewed towards larger values jugding from **quartiles** and **min-max** range